lib/api/apiUtils/object/versioning.js

```diff
@@ -405,6 +405,9 @@ function overwritingVersioning(objMD, metadataStoreParams) {
     metadataStoreParams.lastModifiedDate = objMD['last-modified'];
     metadataStoreParams.updateMicroVersionId = true;
 
+    // set correct originOp
+    metadataStoreParams.originOp = 's3:ObjectRestore:Completed';
```

Comment:
Possible improvement: using consts, from arsenal or an internal file.

Reply:
In all of Cloudserver we set the originOp manually, but i do think ideally we need to use a map of some sort.

Reply:
an enum from typescript would be most appropriate........ once we move to typescript :-/
since the list is in Arsenal, maybe we can define a typescript enum there, and reuse the "generated" variables here (if they exists).

Anyway i agree there is no rush to do it now, it should be a "cleanup" task to do separately and everywhere in cloudserver → @Kerkesni can you create a ticket with the TechDebt label?

Reply:
Done [ZENKO-4409](https://scality.atlassian.net/browse/ZENKO-4409)